drydock_backups/README.rst

```diff
@@ -0,0 +1,46 @@
+Drydock-backuyps
+================
+
+Is a tutor plugin used to easily store backups of mysql and mongodb databases through k8s cronjobs. It backup the databases and stores them in a bucket.
+
+This plugin assumes that the destination bucket is already created and that the credentials to access it are already configured (Works with S3 and Minio buckets indicating his endpoint url through `CUSTOM_STORAGE_ENDPOINT`).
+
+Installation and activation
+---------------------------
+
+Is installed with Drydock, so no need to install it separately.
+
+You can enable it adding `drydock-backups` to the `plugins` section of the `config.yml` file.
+
+Configuration variables
+-----------------------
+
+- **IMAGE**: The image used to run the cronjob. Default: `ednxops/shipyard-utils:v1.0.0`
```

Comment:
Please add the `BACKUP_` prefix for all variables since these are the real names